README.md

```diff
@@ -1,10 +1,81 @@
 # Curriculum
 This project provides study material for [Telegram channel](https://t.me/learniteasy2)
 
-## Draft overview
-* Result of the course: make a working TODO list application.
-  * Success is defined by employed students  
-* Topics:
-  * basics
-  * git basics
-  * 
+## Goal
+Success is defined by employed students
+
+## Plan
+🌟 Season 1: Foundations of IT
+
+	•	Week 1: Introduction to Computers
+	•	Understanding hardware vs. software
+	•	How computers work: CPU, memory, storage
+	•	Week 2: Operating Systems Basics
+	•	Navigating Windows, macOS, and Linux
+	•	Command-line introduction
+	•	Week 3: Networking Fundamentals
+	•	What is the Internet?
+	•	Basic networking concepts and terminology
+	•	Week 4: Cybersecurity Essentials
```

Comment:
I think we can defer the topic of cybersecurity to the future, I have no idea how to explain it at the current stage

Reply:
Addressed!